src/app/dashboard-widgets/environment-widget/environment-widget.component.ts

```diff
@@ -1,16 +1,9 @@
 import { Component, OnInit, ViewEncapsulation } from '@angular/core';
-import { ActivatedRoute } from '@angular/router';
-
-import { BehaviorSubject, ConnectableObservable, Observable, Subject, Subscription } from 'rxjs/Rx';
-
-import { Context, Contexts, Spaces } from 'ngx-fabric8-wit';
-
-import { Environment as ModelEnvironment } from '../../../app/space/create/deployments/models/environment';
-
+import { Contexts, Spaces } from 'ngx-fabric8-wit';
```

Comment:
Did you intend to change this file? It looks like a bad merge as this file does not appear to fit with your changes.